### ISSUE TITLE
Some ARM64 failures need investigation

### ISSUE THREAD

**Issue by TravisEz13** [PowerShell/PowerShell#19047]
Some ARM64 failures need investigation

[screen reader tests fail](https://github.com/PowerShell/PowerShell/pull/19040/files#diff-2b50c5906029d51116f545041ebaa582764af24e58e7c89a09d8ea9deebecf72R47)

[~~DSC tab completion tests don't work on ARM64~~](https://github.com/PowerShell/PowerShell/pull/19040/files#diff-393e1fa171a673ad5e5c3c1f3700ad03d32d46031ffcf72e4eaac1a7c7c46579R2001)
See https://github.com/PowerShell/PowerShell/issues/18781

[notepad is not launched](https://github.com/PowerShell/PowerShell/pull/19040/files#diff-3c801042a6215628d6663af3184c8f0af1574dfe01dd12e4c15ed78cbd767dedR148)

[some experimental features are not enabled](https://github.com/PowerShell/PowerShell/pull/19040/files#diff-ca812dc7ac83139507cab30345879b85295455b3bee8000a1b9810577ea2f617R183)
